.stickler.yml

```diff
@@ -0,0 +1,4 @@
+linters:
+  stylelint:
```

Comment:
You should not be using stylelint for this project. This is a ruby project. Your linter should be rubo cop as seen in the [linter repo](https://github.com/microverseinc/linters-config/tree/master/ruby)
It should look like following:

```yml
linters:
  rubocop:
    display_cop_names: true
files:
  ignore:
    - "bin/*"
    - "db/*"
    - "config/*"
    - "Guardfile"
    - "Rakefile"
```

Reply:
Thank`s for your review! You give good advice about rubo cop.
About git-flow I`m confused cause here https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/57421588/  in the dashboard we have instructions to this task and in part "Remember to:" you can see these two lists: 
-Follow the Github Flow guidelines.
Do not work directly in the master branch. 
-Create a feature-branch branch (give it a descriptive name) for the feature you implement (we assume that this entire project is one feature for learning purposes.) and open a Pull Request from it. When your feature is ready and Pull request approved, you can merge it to the master branch.

Reply:
Yes, so the feature branch should be appropriately named. `develop` branch is used in Git Flow. So your branch could have been named `bubble_sort` etc. and that would be the branch you do the PR on. But this is fine. Just ensure that your feature branch is appropriately named. It is probably best to not name any of your feature branches in future `developer` or `dev` or `development` or similar as a development branch is used in a different way in Git Flow. In GitHub flow only master and feature branches. Does this make sense? Hope it helps 😅 🙂